src/ripple/net/impl/HTTPDownloader.cpp

```diff
@@ -161,36 +166,12 @@ SSLHTTPDownloader::do_session(
     // connection:
     std::uint64_t const rangeStart = size(p);
```

Comment:
This local should be declared after the `connect` call.

Reply:
Done